src/model/Group.js

```diff
@@ -27,9 +27,7 @@ class Group extends Step {
    * @param {object} [config={}] - Group configuration containing group meta information.
    */
   constructor(name, type, config = {}) {
-    if (!(config instanceof Action)) {
-      config.canHavePorts = false;
-    } else {
+    if (config instanceof Action) {
```

Comment:
Just let leave it so. Because I have no suitable solution for this case. Currently it has no sense to create with action